webvtt/api/VTTCue/region.html

```diff
@@ -10,7 +10,7 @@
     document.body.appendChild(video);
 
     var cue = new VTTCue(0, 1, 'text');
-    assert_equals(cue.region, null);
```

Comment:
I'd prefer to have it in there as it more clearly indicates that it should be `null` until line `28`, rather than being set to `null` by adding the cue.